### PR TITLE
Update gazelle and buildifier to latest version

### PR DESCRIPTION
**Pull request by fejta**
/assign @BenTheElder @clarketm @nicoaragon

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,8 +4,8 @@ go 1.13
 
 require (
 	github.com/OpenPeeDeeP/depguard v1.0.1 // indirect
-	github.com/bazelbuild/bazel-gazelle v0.0.0-20181220163313-cdeedbd62467
-	github.com/bazelbuild/buildtools v0.0.0-20180226164855-80c7f0d45d7e
+	github.com/bazelbuild/bazel-gazelle v0.18.2
```
Comment (BenTheElder) [inline go.mod]: whoo no version -> version for gazelle and klog! 😄

Comment (fejta) [reply go.mod]: This silences the build warnings about klog actually being from dims too :)

Comment (dims) [reply go.mod]: /meow

**Pull request merged**